docs: debugging html build (#23443)
gatsbyjs/gatsby: docs/docs/debugging-html-builds.md

1.  You mix up `import` and `require` calls in the same file. This might lead to
    "WebpackError: Invariant Violation: Minified React error #130" [since Webpack 4
    is stricter than v3](/docs/migrating-from-v1-to-v2/#convert-to-either-pure-commonjs-or-pure-es6).

>     The solution is to only use `import` (and be consistent between `gatsby-ssr` and `gatsby-browser`).

Reviewer: ```suggestion
    The solution is to only use `import` this also extends to `gatsby-ssr` and `gatsby-browser` files.
```
